### ISSUE TITLE
 On using ctrl+right key to the code "foo = "Hello world"; the words are not read correctly as seen on screen

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#158909]
**GitHubTags**:#A11ySev2;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code Client;#WCAG1.3.1;#DesktopApp;#BM-VisualStudioCode-Win32-Nov2021;#NVDA;#JAWS;#Win32;#Win11;#A11yeDAD;#External;#External:Chrome;

## Environment Details:​
Application Name: Visual Studio Code
Version: 1.70.2 (user setup)
NVDA2022.2, JAWS 2022.2204.20 ILM

## Repro Steps:

1. Launch Visual Studio Code
2. Navigate to file and press enter select item new file
3. tab to the opened edit dropdown and type HTML and press enter
4. in the editor type the code foo = "Hello world";
5. Run NVDA using ctrl+right key to the code

## Actual:
 On using ctrl+right key to the code "foo = "Hello world"; the words are not read correctly as seen on screen the word foo is read as " "foo =" and  = is also read as "foo equal" and the closing quotes and semi colon is not read on using ctrl +left and right  

## Expected:
The screen reader should read each word on using the ctrl+right key on each use of the key one word should be read that it navigates too

## User Impact:
screen reader users will not get the code read correctly 

## Attachment
![Presentation1](https://user-images.githubusercontent.com/24580570/186180416-92bb43e6-20bb-43a1-bfc0-98d0ac2c6ed2.gif)

**Comment by fsteffi** [comment]
logged in ref to bug https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/182564

**Comment by alexdima** [comment]
cc @isidorn

**Comment by isidorn** [comment]
This is a chrome / nvda bug. I can reproduce on https://google.com
Upstream issue I created some time ago https://bugs.chromium.org/p/chromium/issues/detail?id=1063815&q=reporter%3Ame&can=2
I just pinged our Google contacts

**Comment by fsteffi** [comment]
@isidorn  the issu is resolved on the latest NVDA version and vs code insiders

**Comment by fsteffi** [comment]
closing this bug as bug has been logged to chrome https://bugs.chromium.org/p/chromium/issues/detail?id=1063815&q=reporter%3Ame&can=2

**Comment by isidorn** [comment]
GitHubTags:#Closed;
